tv-app/effecticeTV/app/build.gradle

```diff
@@ -90,6 +91,16 @@ dependencies {
 
     //coil
     implementation libs.coil
+
+    //notion
+    implementation("com.github.seratch:notion-sdk-jvm-core:1.8.0")
```

Comment:
Move all hardcode versions of libraries from build.gradle.kts to [libs.versions.toml](https://docs.gradle.org/current/userguide/platforms.html)